programs/ssl/ssl_client2.c

```diff
@@ -1539,6 +1633,41 @@ int main( int argc, char *argv[] )
                 default: goto usage;
             }
         }
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) 
+
+#if defined(MBEDTLS_ZERO_RTT)
+        else if( strcmp( p, "early_data" ) == 0 )
+        {
+            if( strcmp( q, "disabled" ) == 0 )
```

Comment:
Binary command line flags in `ssl_{client|server}2` are usually enabled/disabled with `0` or `1` instead of `enabled/disabled`.